src/sentry/static/sentry/app/components/onboardingWizard/todoList.jsx

```diff
@@ -0,0 +1,74 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import getOnboardingTasks from 'app/components/onboardingWizard/getOnboardingTasks';
+import SentryTypes from 'app/sentryTypes';
+import TodoItem from 'app/components/onboardingWizard/toDoItem';
```

Comment:
should the file be `todoItem` to match `todoList`?

Reply:
this file already exists in master so leaving it for now